controls/SV-230470.rb

```diff
@@ -55,18 +54,32 @@
   tag gtitle: 'SRG-OS-000062-GPOS-00031'
   tag satisfies: ['SRG-OS-000062-GPOS-00031', 'SRG-OS-000471-GPOS-00215']
   tag gid: 'V-230470'
-  tag rid: 'SV-230470r744006_rule'
+  tag rid: 'SV-230470r1017261_rule'
   tag stig_id: 'RHEL-08-030603'
   tag fix_id: 'F-33114r744005_fix'
   tag cci: ['CCI-000169']
   tag nist: ['AU-12 a']
   tag 'host'
 
-  only_if('This control is Not Applicable to containers', impact: 0.0) {
-    !virtualization.system.eql?('docker')
-  }
+  virtualized_system_no_usb_devices = input('virtualized_system_no_usb_devices')
```

Comment:
```suggestion
  is_virtualized_system_with_no_usb_devices = input('is_virtualized_system_with_no_usb_devices')
```

I prefer a boolean variable read as a true / false statement, like "is_something". If you agree, let's change it throughout the uses of this variable. I will just leave the comment here.

Reply:
Agreed. Fixed.